public/app/features/canvas/elements/server.tsx

```diff
@@ -0,0 +1,474 @@
+              <rect x="55.549" y="71.824" width="107.76" height="5.7472" />
+              <circle
+                className={styles.circle}
+                cx="189.99"
+                cy="68.07"
+                r="9.0052"
+                style={{ animation: bulbAnimation, fill: bulbColor }}
+              />
+            </g>
+          </g>
+        </>
+      ) : data?.type === ServerTypes.Single ? (
+        <>
+          <defs>
+            <clipPath id="serverb">
+              <rect
+                className={styles.pathA}
+                x="77.108"
+                y="24.362"
+                width="115.9"
+                height="197.78"
+                d="M 77.107697,24.361513 H 193.00871 V 222.14392 H 77.107697 Z"
+              />
+              <path
+                className={styles.pathB}
+                d="m26.804 19.62h217.95v206.93h-217.95zm50.304 4.7414v197.78h115.9v-197.78z"
+              />
+            </clipPath>
+          </defs>
+          <g transform="translate(-31.804 -24.362)">
+            <path
+              className={styles.outline}
+              x="36.804028"
+              y="29.620079"
+              width="197.94514"
+              height="186.92975"
+              d="m38.921 29.62h193.71a2.1167 2.1167 45 012.1167 2.1167v182.7a2.1167 2.1167 135 01-2.1167 2.1167h-193.71a2.1167 2.1167 45 01-2.1167-2.1167v-182.7a2.1167 2.1167 135 012.1167-2.1167z"
+              clipPath="url(#serverb)"
+            />
+            <rect x="41.836" y="34.652" width="187.91" height="176.89" style={{ fill: data?.statusColor }} />
+            <g>
+              <path
+                className={styles.server}
+                x="54.86203"
+                y="48.088943"
+                width="159.1676"
+                height="39.961231"
+                d="m56.979 48.089h154.93c1.169 0 2.1167.94768 2.1167 2.1167v145.73c0 1.169-.94768 2.1167-2.1167 2.1167h-154.93c-1.169 0-2.1167-.94768-2.1167-2.1167v-145.73c0-1.169.94768-2.1167 2.1167-2.1167z"
+              />
+              <circle
+                className={styles.circle}
+                cx="189.99"
+                cy="68.07"
+                r="9.0052"
+                style={{ animation: bulbAnimation, fill: bulbColor }}
+              />
+              <rect x="55.556" y="57.472" width="107.76" height="5.7472" />
+              <rect x="55.558" y="86.141" width="107.76" height="5.7472" />
+              <rect transform="translate(31.804,24.362)" x="23.779" y="90.421" width="107.76" height="5.7472" />
+              <g className={styles.thinLine}>
+                <rect x="63.549" y="73.261" width="107.76" height="2.8736" />
+                <rect x="63.574" y="101.9" width="107.76" height="2.8736" />
+                <rect x="63.598" y="130.54" width="107.76" height="2.8736" />
+              </g>
+            </g>
+          </g>
+        </>
+      ) : data?.type === ServerTypes.Database ? (
+        <>
+          {' '}
```

Comment:
```suggestion
```

Reply:
Not even sure why that is there, haha, good catch!